optical/converter/converter.py

```diff
@@ -9,14 +9,14 @@
 import warnings
 from pathlib import Path, PosixPath
 from typing import Dict, List, Optional, Union
-
+from datetime import datetime
 import numpy as np
 import pandas as pd
 import yaml
 from joblib import Parallel, delayed
 from tqdm.auto import tqdm
-
-from .utils import copyfile, ifnone, write_coco_json, write_xml
+import json
```

Comment:
there is a pattern of organizing your imports. it goes like:


=========

import packages that comes with base python e.g., `os`, `pathlib`, `json` etc. 

import third-party packages e.g. `numpy`, `pandas`, `tqdm` etc. 

import modules from the projects e.g. the relative imports

=========
There is a command in vscode in the command palette called "organise imports" which does that automatically for you.